C4/Budgets.pm

```diff
@@ -349,6 +350,11 @@ sub GetBudgetSpent {
     my ($shipmentcost_sum) = $sth->fetchrow_array;
     $sum += $shipmentcost_sum;
 
+    my $adjustments = Koha::InvoiceAdjustments->search({budget_id => $budget_id, closedate => { '!=' => undef } },{ join => 'invoiceid' });
+    while ( my $adj = $adjustments->next ){
```

Comment:
this would be so much nicer if we didn't kill ResultsetColumn in Koha::Objects :(

`->adjustments->sum` would have done it otherwise..